hs_core/models.py

```diff
@@ -3157,6 +3157,7 @@ class BaseResource(Page, AbstractResource):
     objects = PublishedManager()
     public_resources = PublicResourceManager()
     discoverable_resources = DiscoverableResourceManager()
+    _size = models.BigIntegerField(default=0)
```

Comment:
that's a great idea I hadn't considered.  I'll go ahead and code it up.  I'd love to get rid of the async operation.

Reply:
@pkdash @hyi - Any opinions on lazy loading of the size? Something like this?
```
    def size(self):
        if self._size is 0:
            self.calculate_size
        return self._size
```
It means we have to check _size everytime it's returned (not a big deal) and we wouldn't have to necessarily run a management command to populate it (though we may want to anyways to speed things up).  It is possible for a file to have a size of 0, if it's empty, this is an edge case we probably don't care about.

Reply:
@sblack-usu I can see the argument of not needing a separate management command with this extra if logic, but I am thinking perhaps a separate management command would be useful beyond initially populating the sizes, e.g. we may need to run the management command to correct errors of some file sizes triggered by some unexpected actions. That said, if we end up going with this route to add this extra logic, I'd suggest to use ```-1``` as the default value so size of 0 edge case could be treated as a valid case for empty files.

Reply:
great idea @hyi.  I went ahead and removed the management command as it's still written for the older approach.  If we have a specific case to use it for, I'll revivie it, otherwise I'll leave it deleted.  I'll make the update for a default of -1 with a check for <0